lib/rules/default-param-last.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @fileoverview enforce default parameters to be last
+ * @author Chiawen Chen
+ */
+
+"use strict";
+
+module.exports = {
+    meta: {
+        type: "suggestion",
+
+        docs: {
+            description: "enforce default parameters to be last",
+            category: "Best Practices",
+            recommended: false,
+            url: "https://eslint.org/docs/rules/default-param-last"
+        },
+
+        schema: [],
+
+        messages: {
+            shouldBeLast: "Default parameters should be the last of parameters."
```

Comment:
I'm okay with this message, but I wonder if `"Default parameters should be last."` would be just as good.

Reply:
Sounds like an improvement.